themes/rtd_qgis/static/css/qgis_docs.css

```
@@ -49,8 +49,8 @@
 
 /* Avoid images being too big on bigger screen*/
 @media screen and (min-width: 767px) {
-  .rst-content .img {
-    max-width: 70%
+  .rst-content img {
+    max-width: 60%
```

Probably we better fix a size with a value than percentage, avoiding that a screenshot looks bigger in a 767px screen (_767px if 100%_) than in a 900px screen (_540px_) ie 
```suggestion
    max-width: 760px
```
Any html guru to advise? Still, values are picked "randomly". If any best practice exists, will be happy to follow.